java/fury-core/src/test/java/io/fury/serializer/collection/CollectionSerializersTest.java

```diff
@@ -72,7 +71,7 @@ public void testBasicList(boolean referenceTrackingConfig) {
             .withRefTracking(referenceTrackingConfig)
             .requireClassRegistration(false)
             .build();
-    List<String> data = new ArrayList<>(ImmutableList.of("a", "b", "c"));
+    List<String> data = new ArrayList<>(Collections.unmodifiableList(Arrays.asList("a", "b", "c")));
```

Comment:
We may need to use guava list instead, otherwise we can't cover guava collection serialization.

We can ignore all guava usage in test module.

Reply:
Reverted the changes.